Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
@DomT4 Ok, up to you. I'm not wanting to restart and argument either; I'm trying to indicate what would be a more effective communication style here given both of us having better things to do than bicker with one another.